workitem/link/type.go

```diff
@@ -19,7 +19,7 @@ const (
 	// hare are more human-readable.
 	SystemWorkItemLinkTypeBugBlocker       = "Bug blocker"
 	SystemWorkItemLinkPlannerItemRelated   = "Related planner item"
-	SystemWorkItemLinkEpicRelatedUserStory = "Epic Related to User Story"
+	SystemWorkItemLinkEpicRelatedUserStory = "Epic related to User story"
```

Comment:
@kwk Sorry, I thought it's a field name that's why named it like that, but it will  be wrong so changing it.